pkg/apis/v1alpha1/subnet_types.go

```diff
@@ -0,0 +1,78 @@
+/* Copyright © 2021 VMware, Inc. All Rights Reserved.
```

Comment:
rebase the latest code to update the license.

Reply:
Thanks and done.